src/js/components/Video.js

```diff
@@ -228,14 +238,15 @@ export default class Video extends Component {
           {...this._mediaEventProps}>
           {this.props.children}
         </video>
-        {showControls ? this._renderControls() : null}
+        {showControls ? this._renderControls() : undefined}
       </div>
     );
   }
 }
 
 Video.propTypes = {
   colorIndex: PropTypes.string,
+  duration: PropTypes.number,
```

Comment:
can we please add this comment `// remove in 1.0` we will be using it to search for deprecated properties.